doc/_examples/json/MetaTask/OrientationTask.json

```diff
@@ -1,6 +1,6 @@
 {
   "type": "orientation",
-  "body": "bodyName",
+  "body": "r_wrist",
   "orientation": [1.0, 0.0, 0.0,
                   0.0, 1.0, 0.0,
                   0.0, 0.0, 1.0]
```

Comment:
I don't think it's a good idea to describe orientations as matrices in the samples. Notably I remember that Yili hadn't realized that he could use RPY/quaternions instead, and was converting everything to matrices. IMHO we should try to make the samples as-close as possible to how they tend to be used in practice.